.vscode/settings.json

```diff
@@ -4,5 +4,8 @@
     "titleBar.inactiveForeground": "#FFFFFFCC",
     "titleBar.activeBackground": "#0b5d93",
     "titleBar.inactiveBackground": "#0b5d93CC"
-  }
+  },
+  "cSpell.words": [
+    "siteconfig"
```

Comment:
Does this configuration add `siteconfig` to the library of words for [this spellchecking extension](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)? I don't see `siteconfig` in the project anywhere.